File: 2017/spring/src/main/java/com/thetriplet3/af/book/model/Book.java
package com.thetriplet3.af.book.model;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection="books")
public class Book {
    @Id
    private String _id;

    private String name;
    private String isbn;
    private float price;
    private String year;
    private String author;

    public String getName() {
        return name;
    }

    public String getIsbn() {
        return isbn;
    }

    public String getYear() {
        return year;
    }

    public String getAuthor() {
        return author;
    }

    public float getPrice(){
        return price;
    }
    public String getId(){
        return _id;
    }
}

File: 2017/react-client/src/components/Book.js
import React, { Component } from 'react';

class Book extends Component {

    render() {
        return (
            <tr>
                <td>{this.props.book.name}</td>
                <td>{this.props.book.author}</td>
                <td>{this.props.book.price}</td>
                <td>{this.props.book.year}</td>
                <td>{this.props.book.publisher}</td>
                <td>{this.props.book.isbn}</td>
                <td>
                    <button>View</button>
                </td>
                <td>
                    <button>Edit</button>
                </td>
                <td>
                    <button>Delete</button>
                </td>
            </tr>
        );
    }
}

export default Book;